src/main/java/com/microsoft/jenkins/acr/service/AzureContainerRegistry.java

```diff
@@ -42,24 +45,24 @@ public String queueTaskRequest(String resourceGroupName,
                 .withOs(OS.fromString(request.getPlatform().getOs()))
                 .withArchitecture(Architecture.fromString(request.getPlatform().getArchitecture()))
                 .withVariant(Variant.fromString(request.getPlatform().getVariant()));
-        RegistryDockerTaskRunRequest.DefinitionStages.DockerTaskRunRequestStepAttachable attachable = this.getClient()
+        boolean pushable = request.getImageNames() != null && request.getImageNames().size() > 0;
```

Comment:
Simplify with `StringUtils.isNotEmpty()`?

Reply:
imageNames is a list<string>